safehttp/doc.go

```diff
@@ -145,5 +145,17 @@
 //
 // Restricting Risky APIs
 //
-// TODO
+// Security teams work hard to create and enforce security best practices. One of the best
+// practices is avoiding to use packages or function calls that are known to be insecure.
```

Comment:
I'd rephrase this paragraph to focus on the problem, rather than statements about security team and developers.

Reply:
That was replaced, thanks :)